website/addons/wiki/templates/log_templates.mako

```diff
@@ -17,3 +17,27 @@ deleted wiki page
 <a data-bind="attr: {href: wikiUrl}, text: params.page"></a>
 of <a class = "log-node-title-link overflow" data-bind="text:nodeTitle, attr: {href: nodeUrl}"></a>
 </script>
+
+<script type="text/html" id="made_wiki_public">
+made the wiki
+of <a class = "log-node-title-link overflow" data-bind="text:nodeTitle, attr: {href: nodeUrl}"></a>
+publicly editable
+</script>
+
+<script type="text/html" id="made_wiki_public_no_user">
```

Comment:
Are logs in the format `<template>_no_user` used elsewhere?

If not, I'd prefer to keep them in a single template - this should be able to be accomplished by using Knockout to display the correct wording where appropriate.

Splitting them up makes it easier to read the whole message, but also make it easier to update one of them without realizing the other applies as well.